src/tests/topp/THIRDPARTY/third_party_tests.cmake

```diff
@@ -319,10 +319,10 @@ if (NOT (${SIRIUS_BINARY} STREQUAL "SIRIUS_BINARY-NOTFOUND"))
     set_tests_properties("TOPP_SiriusAdapter_2_out_spec" PROPERTIES DEPENDS "TOPP_SiriusAdapter_2")
     
     # test mzML and featureXML without feature_only (filter_by_num_masstraces should automatically set to 1)
-    add_test("TOPP_SiriusAdapter_3" ${TOPP_BIN_PATH}/SiriusAdapter -test -sirius_executable "${SIRIUS_BINARY}" -in ${DATA_DIR_TOPP}/THIRDPARTY/SiriusAdapter_3_input.mzML -in_featureinfo ${DATA_DIR_TOPP}/THIRDPARTY/SiriusAdapter_3_input.featureXML -out_sirius SiriusAdapter_3_output.tmp -preprocessing:filter_by_num_masstraces 3 -sirius:profile qtof -sirius:db all -out_annotated_spectra SiriusAdapter_3_output_spec.tmp -read_sirius_stdout)
+    add_test("TOPP_SiriusAdapter_3" ${TOPP_BIN_PATH}/SiriusAdapter -test -sirius_executable "${SIRIUS_BINARY}" -in ${DATA_DIR_TOPP}/THIRDPARTY/SiriusAdapter_3_input.mzML -in_featureinfo ${DATA_DIR_TOPP}/THIRDPARTY/SiriusAdapter_3_input.featureXML -out_sirius SiriusAdapter_3_output.tmp -preprocessing:filter_by_num_masstraces 3 -sirius:candidates 5 -sirius:profile qtof -sirius:db all -out_annotated_spectra SiriusAdapter_3_output_spec.tmp -read_sirius_stdout)
     add_test("TOPP_SiriusAdapter_3_out" ${DIFF} -in1 SiriusAdapter_3_output.tmp -in2 ${DATA_DIR_TOPP}/THIRDPARTY/SiriusAdapter_3_output.mzTab -whitelist "MTD" "295.18707248574799") # ranks of the compound at "295.18707248574799" m/z shifts depending on OS (unix or windows)
     if (APPLE)  # mzMLs were created on macOS. Since Sirius is nondeterministic we can only test there.
```

Comment:
Ah did you see my note? Maybe we can just test some of the weirdest tests on one platform. Better than whitelisting everything. Either you do a if (LINUX) or you copy the testfile from the mac CI machine.

Reply:
Both Linux and Mac pass now, so either one would work (at least until the next phase of the moon)

Reply:
Set the spec tests 1 and 3 for Linux only, so I'm able to test them locally.

Reply:
Looks like 8 and 9 failed this time on windows.

Reply:
Yes, there are too many exceptions between the operating systems with the tests 8, 9 and 10 so I removed lot's of whitelisting and made them Linux only.

Reply:
Careful. BSD and macOS are Unix flavors. They will also run.

Reply:
Hm, check for OS == "Ubuntu"? It get's determined with `lsb_release -si` in the build scripts.